### PR TITLE
Budget schema

### PR DESCRIPTION
**Pull request by g4s8**
zerocracy/farm#104 - budget xml schema and xsl page

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this pull request

**Comment by 0crat** [comment]
Job `gh:zerocracy/datum#145` is in scope.

File: xsl/pm/cost/budget.xsl [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  <xsl:include href="../../templates.xsl"/>
+  <xsl:template match="/budget">
+    <html lang="en">
+      <body>
+        <section>
+          <h1>
+            <xsl:text>Project budget</xsl:text>
+          </h1>
+          <p>
+            Estimated:
```
Comment (yegor256) [inline xsl/pm/cost/budget.xsl]: @g4s8 use `<xsl:text> and formatting will look better

File: xml/pm/cost/budget/simple.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<budget version="1" updated="2016-12-29T09:03:21.684Z">
+  <estimates>
```
Comment (yegor256) [inline xml/pm/cost/budget/simple.xml]: @g4s8 I would call it `estimate` instead, and add a `job` attribute to it

**Comment by yegor256** [comment]
@g4s8 there are two comments, see above

File: xml/pm/cost/budget/simple.xml (hunk 2 above)
Comment (g4s8) [reply xml/pm/cost/budget/simple.xml]: @yegor256 you meant there should be many `<estimate job="..">` with unique job id inside one `<budget>`?

Comment (yegor256) [reply xml/pm/cost/budget/simple.xml]: @g4s8 yes, exactly. We can estimate one job only once.

Comment (g4s8) [reply xml/pm/cost/budget/simple.xml]: @yegor256 see e76fdd480d2e7a44b6b5354438049875c158d805, it should be fixed

**Comment by 0crat** [comment]
Job `gh:zerocracy/datum#145` assigned to @caarlos0, please go ahead ([policy](http://datum.zerocracy.com/pages/policy.html)).

**Comment by 0crat** [comment]
@caarlos0 this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [par.8](http://datum.zerocracy.com/pages/policy.html#8).

**Comment by 0crat** [comment]
@caarlos0 this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [par.8](http://datum.zerocracy.com/pages/policy.html#8).

**Comment by g4s8** [comment]
@yegor256 please see this PR, your comments were fixed in e76fdd480d2e7a44b6b5354438049875c158d805

**Comment by 0crat** [comment]
Oops! Job `gh:zerocracy/datum#145` is not assigned to anyone.

**Comment by 0crat** [comment]
@caarlos0 resigned from `gh:zerocracy/datum#145`, please stop working.

File: xml/pm/cost/budget/simple.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<budget version="1" updated="2016-12-29T09:03:21.684Z">
+  <estimates id="gh:test/test#1">
```
Comment (yegor256) [inline xml/pm/cost/budget/simple.xml]: @g4s8 should be `estimate` instead, I guess?

**Comment by yegor256** [comment]
@g4s8 see the comment above. Are you sure the build passes?

**Comment by g4s8** [comment]
@yegor256 I forgot to rename it in xsd, this is why `rake` passes. Now it should be correct.

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Oops! There is no order for job `gh:zerocracy/datum#145`.

**Comment by 0crat** [comment]
Job `gh:zerocracy/datum#145` is now out of scope.

**Comment by yegor256** [comment]
@rultor release, tag is `0.45`

**Pull request merged**